### PR TITLE
Set base structure for guidelines

### PR DESCRIPTION
**Pull request by kracekumar**
- Add gudielines directory for all guidelines
- Add top level instructions and structure in README.md

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -5,3 +5,25 @@ Handbook for organizers of PyCon India conference, wittern by the previous organ
 
 http://mail.python.org/pipermail/inpycon/2012-July/005375.html
 
+Contribution guidelines:
+--------
+- All the guidelines should be placed iniside guidelines directory.
```
Comment (vnbang2003) [inline Readme.md]: Spell iniside - inside

Comment (kracekumar) [inline Readme.md]: Fixed.

**Comment by kirang89** [comment]
Maybe create seperate issues for different types of guidelines ?

**Comment by kracekumar** [comment]
Sure, I will do that, meanwhile can you merge ?

**Pull request merged**